Source/gg2/Objects/InGameElements/LooseSheet.events/Collision with BurningProjectile.xml

```diff
@@ -13,13 +13,18 @@
       <not>false</not>
       <appliesTo>.self</appliesTo>
       <arguments>
-        <argument kind="STRING">if(not burning) {
+        <argument kind="STRING">if(!burning
+    and sprite_index != SheetFallingGold1
```

Comment:
Would prefer a "flammable" property, this here feels a bit like determining the action of a button by comparing its label.